File: Payhub - SDK/c#_SDK/PayHubSDK/PayHubSDK/com/payhub/ws/model/Schedule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;

namespace PayHubSDK.com.payhub.ws.model
{
    [DataContract]
    public class Schedule
    {
        [DataMember]
        private string schedule_type;
        public string Schedule_type
        {
            get { return this.schedule_type; }
            set { if (value != null)this.schedule_type = value; }
        }
        [DataMember]
        private int bill_generation_interval;
        public int Bill_generation_interval
        {
            get { return this.bill_generation_interval; }
            set { this.bill_generation_interval = value; }
        }
        [DataMember]
        private ScheduleSartAndEnd schedule_start_and_end;
        public ScheduleSartAndEnd Schedule_start_and_end
        {
            get { return this.schedule_start_and_end; }
            set { if (value != null)this.schedule_start_and_end = value; }
        }
        [DataMember]
        private MontlySchedule monthly_schedule;
        public MontlySchedule Monthly_schedule
        {
            get { return this.monthly_schedule; }
            set { if (value != null)this.monthly_schedule = value; }
        }
        [DataMember]
        private SpecificDatesSchedule specific_dates_schedule;
        public SpecificDatesSchedule Specific_dates_schedule
        {
            get { return this.specific_dates_schedule; }
            set { if (value != null)this.specific_dates_schedule = value; }
        }
        [DataMember]
        private WeeklySchedule weekly_schedule;
        public WeeklySchedule Weekly_schedule
        {
            get { return this.weekly_schedule; }
            set { if (value != null)this.weekly_schedule = value; }
        }
        [DataMember]
        private YearlySchedule yearly_schedule;
        public YearlySchedule Yearly_schedule
        {
            get { return this.yearly_schedule; }
            set { if (value != null)this.yearly_schedule = value; }
        }

        public Schedule(string schedule_type)
        {
            this.schedule_type = schedule_type;
        }
        public Schedule() { }
    }
}

File: Payhub - SDK/java_SDK/PayHubSDK/src/main/java/com/payhub/ws/model/Schedule.java
/**
 * 
 */
package com.payhub.ws.model;

import com.fasterxml.jackson.annotation.JsonInclude;

/**
 * @author agustin
 *
 */
@JsonInclude(JsonInclude.Include.NON_NULL)  
public class Schedule {
	        private String schedule_type;
	        private int bill_generation_interval;
	        private ScheduleSartAndEnd schedule_start_and_end;
	        private MontlySchedule monthly_schedule;
	        private YearlySchedule yearly_schedule;
	        private WeeklySchedule weekly_schedule;
	        private SpecificDatesChedule specific_dates_schedule;
	        
	        public Schedule(){}
	        
	        public Schedule(String schedule_type) { 
	        	this.schedule_type=schedule_type;
	        }
			public String getSchedule_type() {
				return schedule_type;
			}
			public void setSchedule_type(String schedule_type) {
				this.schedule_type = schedule_type;
			}
			public int getBill_generation_interval() {
				return bill_generation_interval;
			}
			public void setBill_generation_interval(int bill_generation_interval) {
				this.bill_generation_interval = bill_generation_interval;
			}
			public ScheduleSartAndEnd getSchedule_start_and_end() {
				return schedule_start_and_end;
			}
			public void setSchedule_start_and_end(ScheduleSartAndEnd schedule_start_and_end) {
				this.schedule_start_and_end = schedule_start_and_end;
			}
			public MontlySchedule getMonthly_schedule() {
				return monthly_schedule;
			}
			public void setMonthly_schedule(MontlySchedule monthly_schedule) {
				this.monthly_schedule = monthly_schedule;
			}
	        
}
